xml/cl.xml

```diff
@@ -6644,46 +6671,47 @@ server's OpenCL/api-docs repository.
                 <type name="cl_unified_shared_memory_type_intel"/>
                 <type name="cl_mem_advice_intel"/>
             </require>
-            <require comment="cl_device_info">
+            <require group="cl_device_info">
                 <enum name="CL_DEVICE_HOST_MEM_CAPABILITIES_INTEL"/>
                 <enum name="CL_DEVICE_DEVICE_MEM_CAPABILITIES_INTEL"/>
                 <enum name="CL_DEVICE_SINGLE_DEVICE_SHARED_MEM_CAPABILITIES_INTEL"/>
                 <enum name="CL_DEVICE_CROSS_DEVICE_SHARED_MEM_CAPABILITIES_INTEL"/>
                 <enum name="CL_DEVICE_SHARED_SYSTEM_MEM_CAPABILITIES_INTEL"/>
             </require>
-            <require comment="cl_unified_shared_memory_capabilities_intel - bitfield">
+            <require group="cl_device_unified_shared_memory_capabilities_intel" etype="bitfield">
                 <enum name="CL_UNIFIED_SHARED_MEMORY_ACCESS_INTEL"/>
                 <enum name="CL_UNIFIED_SHARED_MEMORY_ATOMIC_ACCESS_INTEL"/>
                 <enum name="CL_UNIFIED_SHARED_MEMORY_CONCURRENT_ACCESS_INTEL"/>
                 <enum name="CL_UNIFIED_SHARED_MEMORY_CONCURRENT_ATOMIC_ACCESS_INTEL"/>
             </require>
-            <require comment="cl_mem_properties_intel">
+            <require group="cl_mem_properties_intel">
                 <enum name="CL_MEM_ALLOC_FLAGS_INTEL"/>
             </require>
-            <require comment="cl_mem_alloc_flags_intel - bitfield">
+            <require group="cl_mem_alloc_flags_intel" etype="bitfield">
                 <enum name="CL_MEM_ALLOC_WRITE_COMBINED_INTEL"/>
                 <enum name="CL_MEM_ALLOC_INITIAL_PLACEMENT_DEVICE_INTEL"/>
                 <enum name="CL_MEM_ALLOC_INITIAL_PLACEMENT_HOST_INTEL"/>
             </require>
-            <require comment="cl_mem_alloc_info_intel">
+            <require group="cl_mem_info">
```

Comment:
This should have an "intel" suffix.

```suggestion
            <require group="cl_mem_info_intel">
```

Reply:
I'm going to just commit this, but can you please describe in more detail, why this separation between `cl_mem_info`/`cl_mem_info_intel` (and other suffixed enum type names) exists?

Reply:
Ah, in this case, these 2 are actually different groups of enums for different commands...